pkg/api/types.go

```diff
@@ -3170,9 +3170,11 @@ const (
 	// Local Storage for scratch space, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	// The resource name for ResourceStorageScratch is alpha and it can change across releases.
 	ResourceStorageScratch ResourceName = "storage.kubernetes.io/scratch"
```

Comment:
These resources need to be removed. @NickrenREN @jingxu97 can either you clean that up?